retroshare-gui/src/gui/gxschannels/GxsChannelDialog.cpp

```diff
@@ -64,8 +64,8 @@ void GxsChannelDialog::handleEvent_main_thread(std::shared_ptr<const RsEvent> ev
 
         switch(e->mChannelEventCode)
         {
-		case RsChannelEventCode::NEW_MESSAGE:
-		case RsChannelEventCode::UPDATED_MESSAGE:        // [[fallthrough]];
+		case RsChannelEventCode::NEW_MESSAGE:             // [[fallthrough]];
+		case RsChannelEventCode::UPDATED_MESSAGE:
```

Comment:
AFAIU `// [[fallthrough]];` should be added here too

Reply:
fixed